packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -185,6 +185,23 @@ - (void)onMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
       [_markersController removeMarkerIds:markerIdsToRemove];
     }
     result(nil);
+  } else if ([call.method isEqualToString:@"markers#showInfoWindow"]) {
+    id markerId = call.arguments[@"markerId"];
+    if ([markerId isKindOfClass:[NSString class]]) {
+      [_markersController showMarkerInfoWindow:markerId];
+    }
+    result(nil);
+  } else if ([call.method isEqualToString:@"markers#hideInfoWindow"]) {
+    id markerId = call.arguments[@"markerId"];
+    if ([markerId isKindOfClass:[NSString class]]) {
+      [_markersController hideMarkerInfoWindow:markerId];
+    }
+    result(nil);
+  } else if ([call.method isEqualToString:@"markers#isInfoWindowShown"]) {
+    id markerId = call.arguments[@"markerId"];
+    if ([markerId isKindOfClass:[NSString class]]) {
+      result([_markersController isMarkerInfoWindowShown:markerId]);
```

Comment:
A compile error occurs at this line . 

Please replace this line with

```
result([NSNumber numberWithBool:[_markersController isMarkerInfoWindowShown:markerId]]);
```

Reply:
should i just copy paste that or do you want to make a pr on my repo?